app/assets/javascripts/shared/SharedService.js.coffee

```diff
@@ -27,7 +27,7 @@ SharedService = ($http, $state, $window, $document) ->
   Service.getLanguageCode = (langName) ->
     # will take "English" and return "en", for example
     if langName
-      _.invert(Service.languageMap)[_.startCase(langName)]
+      _.invert(Service.languageMap)[_.startCase(_.lowerCase(langName))]
```

Comment:
@software-project why did this update need to be made? seems weird to lower case then start case?

Reply:
It seems like it used to work differently with the past version, as tests were failing for upper case scenarios. That fixed it.

Reply:
oh I see. There was a change to startCase to just capitalize the first char rather than changing to capitalization case. 

e.g. 
used to do EN -> En, now just does EN -> EN.

Maybe we could replace with [Capitalize](https://lodash.com/docs/4.17.11#capitalize) Instead and see if the tests pass? Seems to be what we're interested in

Reply:
Yes, exactly. That should also work, will give it a try.